src/momento/cache_client_async.py

```diff
@@ -186,6 +190,45 @@ def __init__(self, configuration: Configuration, credential_provider: Credential
             for _ in range(CacheClientAsync._NUM_CLIENTS)
         ]
 
+    @staticmethod
+    async def create(
```

Comment:
Is this how we've called this in other libraries?

Reply:
Yes!